docs/block-elements.md

```diff
@@ -633,6 +633,22 @@ As same as `<Input type="hidden">`, `submit` prop in `<Modal>` must not define w
 
 It's exactly same as [`<Input>` component](#input-props), except `type` prop.
 
+## <a name="mrkdwn_component" id="mrkdwn_component"></a> [`<Mrkdwn>`: Markdown Composition Component](https://api.slack.com/reference/block-kit/composition-objects#text)
+
+Mrkdwn is a text composition component is used when its needed to have a Text object and explicitly set the `verbatim` property. Setting `verbatim` to false will tell Slack to auto-convert links, conversaion names, and certain mentions to be linkified and automatically parsed. If `verbatim` set to true Slack will skip any prepreprocessing.
```

Comment:
I can understand the use case well, but as our position, we cannot recommend using `verbatim=false` to enable automatic parsing because [Slack is making clear a possibility of deprecating gradually in future](https://api.slack.com/reference/surfaces/formatting#why_you_should_consider_disabling_automatic_parsing).

We think `verbatim` attribute is like an escape hatch for the traditional Slack app. jsx-slack has already take `verbatim=true` as a default option even if not using `<Mrkdwn>` composition object. So we don't want to use this by users who will create new app with jsx-slack.

Thus, we should make clear these intents within this document.